pages/index.tsx

```diff
@@ -1,70 +1,70 @@
+import '@reach/dialog/styles.css'
 import type { NextPage } from 'next'
-import Head from 'next/head'
-import Image from 'next/image'
 import styles from '../styles/Home.module.css'
+import {Dialog} from '@reach/dialog'
+import { useState } from 'react'
 
-const Home: NextPage = () => {
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
+function LoginForm({onSubmit, buttonText}){
```

Comment:
You might want to install `Prettier` + `ESlint` in your `vscode` then update your settings to `format on save` It will help you to automatically fix linting errors such as no space between `{}`, e.g. `LoginForm({ onSubmit, buttonText})`, `import { Dialog } from '@reach/dialog'`

Reply:
Usually, we set an interface to specify which types are needed.

```suggestion
interface LoginFormProps {
  onSubmit: () => void
  buttonText: string
}

export const LoginForm({ onSubmit, buttonText }: LoginFormProps) => {
```